rails-install-ubuntu.sh

```diff
@@ -2,50 +2,69 @@
 
 set -e
 
+echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
+sudo add-apt-repository ppa:chris-lea/node.js
```

Comment:
Huh, I forgot there was an updated version. If you take a look at that script you linked, it actually removes this PPA if present, because this was the official one before they made their own `deb` repository. I'll flip it around to that.